src/repository/screens/repository.screen.js

```diff
@@ -145,14 +161,16 @@ class Repository extends Component {
     const openIssues = pureIssues.filter(issue => issue.state === 'open');
     const closedIssues = pureIssues.filter(issue => issue.state === 'closed');
 
-    const repositoryActions = [starred ? '★ Unstar' : '★ Star'];
+    const repositoryActions = [starred ? 'Unstar' : 'Star'];
     const showFork =
       repository && repository.owner && repository.owner.login !== username;
 
     if (showFork) {
       repositoryActions.push('Fork');
     }
 
+    repositoryActions.push('Share');
```

Comment:
Yea true, i was for now keeping it as it was. But this looks good! Will update it.